components/progressbar/stories/progressbar.test.js

```diff
@@ -15,11 +15,19 @@ export const ProgressBarGroup = Variants({
 			testHeading: "Text overflow",
 			label: "Storage space remaining for XYZ user"
 		},
+		/* The gradient story below supports linear-gradients used by Express. For use cases that require a custom 
+		linear-gradient for any --mod-*-{fill} properties, set those custom properties in CSS.
+		*/
+		{
+			testHeading: "Gradient support",
+			trackFill: "linear-gradient(to right, hotpink, orange)",
+			progressBarFill: "linear-gradient(to left, teal, purple)",
+		},
 	],
 	stateData: [
 		{
 			testHeading: "Indeterminate",
-			indeterminate: true,
+			isIndeterminate: true,
```

Comment:
Should I have _not_ corrected this arg? Is there a reason for us to keep it "indeterminate" as opposed to the "isIndeterminate" arg spelling?

Reply:
`isIndeterminate` seems perfect to me because it's a state